mllib/src/main/scala/org/apache/spark/ml/Pipeline.scala

```diff
@@ -301,9 +308,11 @@ class PipelineModel private[ml] (
   }
 
   @Since("2.0.0")
-  override def transform(dataset: Dataset[_]): DataFrame = {
+  override def transform(dataset: Dataset[_]): DataFrame = MLEvents.withTransformEvent(
+      this, dataset, logging = true) {
```

Comment:
I manually tested the logging and this logging way is borrowed from `transformSchema`.